config/application.rb

```diff
@@ -26,5 +26,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    config.serve_static_assets = true
```

Comment:
You shouldn't need to set this to true. If it doesn't work when you deploy to Heroku let me know.